src/css/variablesCustom.css

```diff
@@ -180,82 +180,13 @@
   --footer-default-footer-margin: 0 auto;
   --footer-default-footer-width-mobile: 100%;
   --footer-default-footer-margin-mobile: 0;
-  --p-margin: 0 0 calc(16rem/18) 0; /* 16px */
-  --ul-margin: 0 0 calc(16rem/18) calc(24rem/18);
+  --p-margin: 0 auto calc(16rem/18); /* 16px */
+  --ul-margin: 0 auto calc(16rem/18);
   --ul-li-padding-left: calc(4rem/18);
   --h-hyphens: auto;
 
-  /* Headlines */
```

Comment:
thomas is going to fix all the heading and font stuff from mdx on a higher level aka. sys.css. sorry, for the trouble.